src/main.rs

```diff
@@ -99,6 +100,11 @@ impl Termination for MainExit {
                 eprintln!("{err:?}");
                 ExitCode::from(16)
             }
+            Self::JoinErr(err) => {
+                error!("Fatal error:");
+                eprintln!("{err:?}");
+                ExitCode::from(16)
```

Comment:
Let's define another code for this like 17, might as well.

Reply:
Done!